src/ai/susi/server/api/aaa/GetAllUsers.java

```diff
@@ -0,0 +1,62 @@
+package ai.susi.server.api.aaa;
+
+import ai.susi.DAO;
+import ai.susi.json.JsonObjectWithDefault;
+import ai.susi.server.*;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Created by chetankaushik on 31/05/17.
+ */
+public class GetAllUsers extends AbstractAPIHandler implements APIHandler {
+
+    private static final long serialVersionUID = 4538304346942632187L;
+
+    @Override
+    public String getAPIPath() {
+        return "/aaa/getAllUsers.json";
+    }
+
+    @Override
+    public BaseUserRole getMinimalBaseUserRole() {
+        return BaseUserRole.ADMIN;
+    }
+
+    @Override
+    public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
+        return null;
+    }
+
+    @Override
+    public JSONObject serviceImpl(Query call, HttpServletResponse response, Authorization rights, final JsonObjectWithDefault permissions)
```

Comment:
Should there be some functionality for applying a filter (e.g. search) or for limiting the result? Pagination?

As long as we're having not too many users this should be fine.